.github/workflows/ci.yaml

```diff
@@ -136,6 +136,9 @@ jobs:
           - connector: source-hubspot
             connector_type: capture
             python: true
+          - connector: source-salesforce
```

Comment:
Please add to the newer .github/workflows/python.yaml -- this workflow can be considered "deprecated", we're trying to break it up into smaller, more-targeted workflows.